runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -47,6 +47,21 @@
 	 */
 	public static final int RUN_SLOWED_DEPLETION_ACTIVE = 25;
 
+	/**
+	 * Stamina effect timer
+	 * Number of game tick remaining on stamina effect in intervals of 10; for a value X there are 10 * X game ticks remaining.
```

Comment:
typo, I'll commit via GH's interface
```suggestion
	 * Number of game ticks remaining on stamina effect in intervals of 10; for a value X there are 10 * X game ticks remaining.
```